recipes/torch-frft/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "torch-frft" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torch_frft-{{ version }}.tar.gz
+  sha256: 9926b92437831c8522028f7da90334816eafba5ad303c51eaf85f9f505fe2fda
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10,<3.12
+    - poetry-core
+    - pip
+  run:
+    - python >=3.10,<3.12
+    - numpy >=1.24.3,<2.0.0
+    - scipy >=1.10.1,<2.0.0
+    - pytorch >=2.0.0,<2.0.1
+    - __unix
```

Comment:
Id the reason for this is the lack of pytorch on Windows, you don't need it. The package won't be installable on Windows as anyway and, if pytorch is ever made available there, this package would be ready. With the `__unix` selector it won't.

```suggestion
```

Reply:
It is exactly like what you've described, thank you for the suggestion 👍🏻